src/styles/BasePage.scss

```diff
@@ -2,7 +2,7 @@
   font-weight: 500;
   font-family: Raleway, sans-serif !important;
   font-size: 2.5rem;
-  margin-bottom: 1.5rem;
+  margin-bottom: 1rem;
```

Comment:
I suspect this is causing margin issues.

Reply:
This is only affecting the page-title so sadly  not